pcsx2/FPU.cpp

```diff
@@ -180,14 +188,14 @@ void ABS_S() {
 
 void ADD_S() {
 	_FdValf_  = fpuDouble( _FsValUl_ ) + fpuDouble( _FtValUl_ );
-	checkOverflow( _FdValUl_, FPUflagO | FPUflagSO, 1 );
-	checkUnderflow( _FdValUl_, FPUflagU | FPUflagSU, 1 );
+	if (checkOverflow( _FdValUl_, FPUflagO | FPUflagSO)) return;
+	if (checkUnderflow( _FdValUl_, FPUflagU | FPUflagSU)) return;
```

Comment:
Yes, you're right. I did a search and replace. I like the symetry but I would remove them if you prefer.